docs/encyclopedia/application-message-passing.mdx

```diff
@@ -220,22 +220,40 @@ Every time the Workflow wakes up--generally, it wakes up when it needs to--it wi
 
 This execution is on a single thread–while this means you don’t have to worry about parallelism, you do need to worry about concurrency if you have written Signal and Update handlers that can block. These can run interleaved with the main Workflow and with one another, resulting in potential race conditions. These methods should be made reentrant.
 
+#### Initializing the Workflow first {#workflow-initializers}
+
+Initialize your Workflow's state before handling messages. This prevents reading uninitialized instance variables.
```

Comment:
```suggestion
Initialize your Workflow's state before handling messages.
This prevents reading uninitialized instance variables.
```

Reply:
Suggestion, add actor (This prevents your message handlers) so it's clear who is reading.
Suggestion, add consequence -- what might happen when instance variables are uninitialized and therefore nonsense values.

Reply:
I incorporated the first point.  Second point I think is sufficiently clear to programmers, and also the exact nature depends on the language.